tests/test_config.py

```diff
@@ -36,7 +36,7 @@ def git_init(message="hello world") -> None:
 
 
 def test_config_data_is_loaded_from_file() -> None:
-    tpl = Template(str(Path("tests", "demo_data")))
+    tpl = Template("tests/demo_data")
```

Comment:
I've reverted these kinds of changes because I think Copier handles POSIX paths fine both on Unix and Windows due to `pathlib.Path()` usage.

I tested this with `pathlib`:

```python
>>> pathlib.PureWindowsPath("a/b").parts
('a', 'b')
```

And `pathlib.Path()` seems to delegate the OS-specific implementation to the respective classes.

Reply:
let's undo these too please.